### PR TITLE
Music & Sound volume sliders in the settings menu

### PR DESCRIPTION
**Pull request by Yoshie2000**
This feature is highly requested according to the Trello board. This is my first (and arguably rather simple) contribution to shapez.io after I had been playing around with the code for a bit.

I already added translations for English and German, I also tried my best to adapt to the existing code style, and I feel like I did pretty well.

### CODE REVIEW

File: src/js/platform/sound.js [hunk 1]
```diff
@@ -202,6 +211,41 @@ export class SoundInterface {
...
+    getSoundVolume() {
+        return this.soundVolume;
+    }
+
+    /**
+     * Sets the music volume
+     * @param {number} volume
+     */
+    setMusicVolume(volume) {
+        this.musicVolume = clamp(volume, 0, 1);;
```
Comment (tobspr) [inline src/js/platform/sound.js:235]: Double semicolon here, but i can fix that

**Comment by tobspr** [comment]
Thats pretty cool! One issue though, please make sure you use the same prettier config (I think I checked it in, did I?) so the PR doesn't get messed up with formatting stuff

I will have a look

**Comment by Yoshie2000** [comment]
Oh yeah, sorry for that, I fixed the semicolon and the formatting.

**Comment by tobspr** [comment]
I think it still has a conflict, I guess because I added a new setting in the meantime, could you have a look? :)

**Comment by Yoshie2000** [comment]
For whatever reason there were a few linter errors in files where I didn't change anything, but all tests have passed now

**Comment by tobspr** [comment]
yeah thats a new github feature ....

**Comment by tobspr** [comment]
Ty!

File: translations/base-de.yaml [hunk 2]
```diff
@@ -728,6 +728,16 @@ settings:
             description: >-
                 Bei der  Aktivierung wird die Musik stummgeschaltet.
 
+        soundVolume:
+            title: Geräuschlautstärke
+            description: >-
+                Ändert die Lautstärke von Geräuschen.
```
Comment (DerFeldspatz) [inline translations/base-de.yaml:734]: What about `Regelt` or `Regler für` instead of  `Ändert`

**Comment by squeek502** [comment]
Note that after updating to latest master after this was merged, I'm getting the error:

```
Failed to read app_settings.bin: invalid-data: Bad setting value for soundVolume: undefined
```

**Pull request merged**